### ISSUE TITLE
NOT DONE Add 2 progress bars for writing a PDF and wrinting png files in Publish()

### ISSUE THREAD

**Issue by epogrebnyak** [epogrebnyak/rosstat-kep-data#112]
Todos are in publish.py https://github.com/epogrebnyak/rosstat-kep-data/blob/master/kep/extract/publish.py#L62-L73

**Comment by epogrebnyak** [comment]
https://github.com/epogrebnyak/rosstat-kep-data/blob/master/notes/issues/progress_bar_sample.py

**Comment by epogrebnyak** [comment]
Todos are in publish.py c3800c8

**Comment by epogrebnyak** [comment]
@sinancetinkaya:  I ran `python update.py` in project root and there is no progress bar. Also changed import statement and filename. Please update. 

Screen output:

```
Database updated from:
    C:\Users\Евгений\Documents\GitHub\bis\rosstat-kep-data\data\2015\ind12
New CSV dumps created.

Writing Excel files...
Writing markdown file with variable names...
Writing PDF...
Writing 129 PNG files...
Output located at: C:\Users\Евгений\Documents\GitHub\bis\rosstat-kep-data\output
```

**Comment by sinancetinkaya** [comment]
file name has been changed, this statement should work.
from kep.extract.local_progressbar import progressbar

**Comment by epogrebnyak** [comment]
import is not a proplem, it was [fixed](https://github.com/epogrebnyak/rosstat-kep-data/blob/test/kep/extract/publish.py#L11), but there is still no progress bar when running `python update.py`

**Comment by sinancetinkaya** [comment]
Forgot to change "infinite" parameter to "True"

**Comment by epogrebnyak** [comment]
@sinancetinkaya Sinan, please - do an update with the commit, it is hard to follow the way you do it and smiley is not fun when I spend time tracking your bugs instead of you solving mine.

Above would have been better as:

"infinite" parameter to set to "True" in 2775e9a2e7a97d28402ed345b70eb1f913d0d288

**Comment by epogrebnyak** [comment]
Still same output for me and no progress bar for me.
Also note TODO2  is known length progress bar, but it likely must wrap plotting function in plots.py
